components/TableRow/TableRow.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+  <div class="item-background">
+    <p class="container-header">{{ title }}</p>
+    <ul class="container flex">
+      <li class="item flex-item" v-for="(distribute,i) in distributions" :key="i">
+        <slot :item="distribute">          
+          <div class="circle margin-20" :class="distribute.name"></div>
+          <div class="margin-20 bold">{{distribute.name | capitalize}}</div>
+          <div class="margin-20">
+            <label class="label">{{distribute | label(distribute.amount) | capitalize}}:</label>
+            <label class="value">{{distribute.amount}}</label>
+          </div>
+          <div class="margin-20">
+            <label class="label">{{distribute | label(distribute.percentage) | capitalize}}:</label>
+            <label class="value">{{distribute.percentage}}</label>
+          </div>
+        </slot>
+      </li>
+    </ul>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "TableRow",
```

Comment:
This component is not a table and row and it's also not generic at all so this should be done at the application level, not on the library... this component should be removed

Reply:
Removed.